tests/step_implementers/create_container_image/test_buildah.py

```diff
@@ -476,8 +487,10 @@ def test_create_container_image_specify_buildah_implementer_with_destination_val
                 '--layers',
                 '-f', file,
                 '-t', tag,
+                '--authfile', StringRegexParam(r".*/.containers/auth.json"),
```

Comment:
I have a feeling this may bite us.

Reply:
its a test. and the other way of doing checks of parameters on calls is to basically re-rewite the logic of the assert_called_with (i can find the stackoverflow link if you want). so i was like f that noise, and wrote this. https://xkcd.com/208/